### ISSUE TITLE
Transfer SUDT doesn't support CKB2021 address

### ISSUE THREAD

**Issue by e00dan** [ckb-js/ckit#99]
When using:
```
const txBuilder = new AcpTransferSudtBuilder(
      { recipients: [{ recipient: toAddress, amount, sudt: this.createSUDTTypeScript(issuerLockHash), policy: "createCell" }] },
      this._provider,
      sender
    );
```

CKB2021 isn't supported. Error (invalid checksum):
<img width="1800" alt="image" src="https://user-images.githubusercontent.com/4950658/160881487-d0c60474-d4eb-46e3-8a38-4c5c0f129931.png">

Broken (ckb2021 address):
[ckt1qpvvtay34wndv9nckl8hah6fzzcltcqwcrx79apwp2a5lkd07fdxxqw3wvcn550clsmmean4dx6x827cnkqcgncz88uxh](https://explorer.nervos.org/aggron/address/ckt1qpvvtay34wndv9nckl8hah6fzzcltcqwcrx79apwp2a5lkd07fdxxqw3wvcn550clsmmean4dx6x827cnkqcgncz88uxh)

Works (old, pre2021 address):

[ckt1q3vvtay34wndv9nckl8hah6fzzcltcqwcrx79apwp2a5lkd07fdx85tnxya9r78ux770vatfk336hkyasxzy7r38glc](https://explorer.nervos.org/aggron/address/ckt1q3vvtay34wndv9nckl8hah6fzzcltcqwcrx79apwp2a5lkd07fdx85tnxya9r78ux770vatfk336hkyasxzy7r38glc)

cc @homura @mkxbl

**Comment by homura** [comment]
thx for your reporting, we will upgrade lumos@0.17.0-rc9 to support CKB2021 address

**Comment by homura** [comment]
hi, we have released [v0.2.0](https://github.com/nervosnetwork/ckit/pull/101) to support [CKB2021 address](https://github.com/nervosnetwork/ckit/pull/100)

```ts
const txBuilder = new AcpTransferSudtBuilder(
  {
    recipients: [
      {
        // both CKB2021 and CKB2019 address are supported 
        recipient: toAddress, 
        amount,
        sudt: this.createSUDTTypeScript(issuerLockHash),
        policy: 'createCell',
      },
    ],
  },
  this._provider,
  sender,
);

```

**Comment by e00dan** [comment]
Fixed. Thanks to you this library: https://github.com/Roger-RumbleFish/nervos-godwoken-integration has now support for sending CKB/SUDT using Ckit. : )
